src/org/sosy_lab/cpachecker/cfa/parser/llvm/CFABuilder.java

```diff
@@ -1325,6 +1340,17 @@ private int getLength(Value pAggregateValue) {
       throws LLVMException {
     long assigneeId = pAssignee.getAddress();
     CType expectedType = pAssignment.getExpressionType();
+
+    // the left hand side is not a variable but some constant expression
+    // (e.g. dereference of a pointer to a global variable)
+    if (pAssignee.isConstantExpr()) {
+        CExpression assigneeExpr = getExpression(pAssignee, expectedType, pFileName);
+        return ImmutableList.of(
+            new CExpressionAssignmentStatement(
+                getLocation(pAssignee, pFileName),
+                (CLeftHandSide)assigneeExpr, (CExpression) pAssignment));
```

Comment:
This is very finnicky, but I would do consistent spacing (or non-spacing) between the cast type and the variable.

Reply:
Ok.